### DISCUSSION TITLE
How to disable the thumbnail image on Windows notifications?

### DISCUSSION THREAD

**Discussion by sophisma** [aclap-dev/video-downloadhelper#691] [Help and Q&A] [answered]
### Your question

Hi. 
On the latest version we now get a thumbnail image, besides the text, on the Windows notifications.
I want to get the notification text but disable the thumbnail.
I've searched the settings to see if it could be done but I'm unnable to find it.
Does anyone know if this is possible to do?

### Addon & CoApp version and details

Add-on Version 9.0.1.13 (stable).
CoApp Version: v2.0.19

**Comment by paulrouget** [comment]
Ah it's not possible for now. But I can add an option if you want.

**Reply by sophisma** [reply]
That would be nice.
Meanwhile I reverted back to the legacy UI.
One thing I also want to mention is that on the new UI the settings are more limited than on the older ones, a bunch of things are missing, on the legacy UI you can customize a lot more stuff.
I don't know if you are moving in a different direction but honestly having the power to change some things is nice.

**Reply by paulrouget** [reply]
What things are you missing? Because most option in the old UI were actually not working (just options from a very ancient version of VDH)

**Reply by paulrouget** [reply]
> That would be nice.

I'll make it happen :)

**Reply by sophisma** [reply]
> What things are you missing? Because most option in the old UI were actually not working (just options from a very ancient version of VDH)

I don't have it installed on this computer but, from the top of my head, things like the filename size, ignore files under a certain size and I believe there was another option regarding file sizes over a certain value.

**Reply by sophisma** [reply]
> > That would be nice.
> 
> I'll make it happen :)

Thank you.

**Reply by paulrouget** [reply, marked as answer]
This is going to be part of v9.0.2.7 - it should be available in a few days (it will auto update).
